csrc/scheduler/mma_utils.h

```diff
@@ -261,6 +289,9 @@ using DependenciesMap = std::map<TensorView*, DomainsDesc>;
 //!  transposition of inputs in mma instructions, while other (e.g. Turing,
 //!  Ampere) the only supported transposition is TN which means that mma
 //!  instruction first input is transposed, the second input is non-transposed.
+MatmulProblemLayoutOpt getMmaLayout(
```

Comment:
Please mention that the other `getMmaLayout` is just a wrapper of this `getMmaLayout` for convenience.